src/gui/menu_list.cpp

```diff
@@ -0,0 +1,24 @@
+#include "gui/menu_item.hpp"
+#include "gui/menu_list.hpp"
+#include "gui/menu_manager.hpp"
+
+ListMenu::ListMenu(std::vector<std::string>* items_, int* selected_) : 
```

Comment:
Why are the arguments taken as pointer? Can they be null?

Reply:
I don't think the pointer is necessary. I'll fix that.